plugins/microservices/src/test_msp_functions.cpp

```diff
@@ -0,0 +1,71 @@
+/// \file
+
+#include "irods/irods_ms_plugin.hpp"
+#include "irods/msParam.h"
+
+#include "msi_assertion_macros.hpp"
+
+namespace
+{
+    auto msi_impl(RuleExecInfo* _rei) -> int
+    {
+        IRODS_MSI_TEST_BEGIN("testing msp_to_ function: msp_to_string")
```

Comment:
Tell me about the name of the file and the name of the test?

Is the plan to add more `msp` functions to this file over time?

Reply:
Yep, that was the original idea, though I'm not sure how watertight it is. Terrell mentioned having a family of these, potentially, so I thought I could consider doing that.
If that's a bad idea though, I'm not against just making this test for msp_to_string only!